README.md

```diff
@@ -123,6 +131,31 @@ A systemd unit is also provided to run the crawler on repeat. Simply run
 `systemctl start crawler@<your username>.service` to start the crawler running
 on repeat.
 
+### Using PostgreSQL for data storage and queries
+
+The data collection programs - the sorter and crawler - are integrated with a PostgreSQL database. When the `use_database` option is set to True in the sorter part of `config.ini`, the sorter will save its sorted onion addresses in the database. When the `use_database` option is set to True in the crawler part of `config.ini`, the crawler will grab onions from the database, connect to them, measure traces and store them back in the database. In order to use these features, the following PostgreSQL environmental variables must be defined: 
+
+```
```

Comment:
The user should not have to set these. Instead we can export them in `/etc/profile` or `/etc/profile.d/`. Also, `PGPASSWORD` should be avoided in favor of `PGPASSFILE`.